source/admin_manual/list_of_events.rst

```diff
@@ -812,10 +812,17 @@ IMAP command
 |                     |                                                      |
 |                     | .. versionadded:: v2.3.9                             |
 +---------------------+------------------------------------------------------+
-| cmd_name            | IMAP command name (e.g. ``FETCH``)                   |
+| cmd_name            | IMAP command name uppercased (e.g. ``FETCH``).       |
+|                     | Since v2.3.11 contains "unknown" for unknown command |
+|                     | names.                                               |
```

Comment:
iirc there is `.. versionchanged: ` too.

Reply:
Yeah. Now it looks like:

```
IMAP command name uppercased (e.g. FETCH).

New in version v2.3.9.

Changed in version v2.3.11: Contains “unknown” for unknown command names.
```

Reply:
Actually, I'll remove quotes from "unknown" also and make it monospace.